portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,33 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div class="cover">
```

Comment:
Is the class used here? I'm not very familiar with HTML, but I gather the class attribute is usually used to [refer to a class in a style sheet or javascript file](https://www.w3schools.com/tags/att_class.asp).

If you haven't created a corresponding class in this PR, it might be better to remove this. Or you could add the class in this PR.

Reply:
Yes, the class is being used here in the style sheet. I'll make sure to add the corresponding classes in the PR. Thanks for bringing this up.